index.html

```diff
@@ -0,0 +1,136 @@
+
+<!DOCTYPE html>
+<html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <!-- link to CSS -->
+    <link rel="stylesheet" href="stylesheet.css">
+  <!--link to fontawesome icons  -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/9f86628c0e.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Title of document -->
+    <title>Webpage</title>
+    <header>
+      <!-- Navbar -->
+        <div id="topnav">
+          <!-- Home button and icon -->
+          <a href="#home"><i class="fa fa-fw fa-home" alt ="House symbol"></i> Home</a>
+           <!-- About me button -->
+            <a href="#about-me">About</a>
+            <!-- SOC button and icon -->
+            <a href="#SOC"><i class="fas fa-desktop"></i> SOC</a>
+            <!-- Contact button and icon -->
+            <a href="#contact">Contact</a>
+          </div>
+      </header>
+</head>
+<body>
+  <section class="home" id= "home">
+    <img src="src/efe-kurnaz-RnCPiXixooY-unsplash.png" class="home_img" alt= "Background of car"/>
+  </section>
+  <section>
+<!-- About me section -->
+    <section class="about" id= "about-me">
+      <div class= "about-container"> 
```

Comment:
There are a few instances of inconsistent formatting. Use a linter such as Prettier to ensure formatting is consistent throughout, and provide an example of best practice to aim for.

```suggestion
      <div class="about-container"> 
```